apps/api/package.json

```
@@ -32,6 +32,7 @@
     "@nestjs/core": "9.0.2",
     "@nestjs/graphql": "10.0.18",
     "@nestjs/jwt": "9.0.0",
+    "@novu/bulkSms": "^0.8.0",
```

```suggestion
    "@novu/bulkSms": "^0.9.0",
```
Also if this was added with Nx or PNPM this should be alphabetically ordered in the dependencies list.

---

ok